tests/aarch64/checker/checker.c

```diff
@@ -0,0 +1,305 @@
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
```suggestion
#include <string.h>
#include <stdint.h>
```

Reply:
No longer relevant after the switch to unicorn.